Formula/terminalimageviewer.rb

```diff
@@ -0,0 +1,25 @@
+class Terminalimageviewer < Formula
+  desc "Small program to display images in a terminal using RGB ANSI codes and unicode block graphics characters"
+  homepage "https://github.com/stefanhaustein/TerminalImageViewer"
+  url "https://github.com/stefanhaustein/TerminalImageViewer/archive/v1.0.0.zip"
+  sha256 "ab44f7d7b26152590a929fec6586e33a3b47737426467fc02c5d5149593ba094"
+  head "https://github.com/stefanhaustein/TerminalImageViewer.git"
+  depends_on "imagemagick"
+  depends_on "gcc" => :build unless OS.mac?
+  def install
+    cd "src/main/cpp" do
+      system "make"
+      bin.install "tiv"
+    end
+  end
+
+  def caveats
```

Comment:
This is the standard binary name so it doesn't need to be a caveat.

Reply:
So, I can just remove the entire caveat and do nothing else?

Reply:
Yes